pipeline/mrms_download.py

```diff
@@ -4,16 +4,23 @@
 from urllib.request import urlopen
 
 
-def download(inventory, dest_dir, max_download=4):
-    """Given an inventory file for a particular day, and a destination
-    directory, download all files in the inventory that are not
+def download(inventory, dest_dir = os.path.join(os.getcwd(),'Data'), max_download=4):
+    """Given an inventory file (or dataframe as returned by inventory) 
+    for a particular day, and a destination directory (default to current 
```

Comment:
Ah... so you are trying to make this function polymorphic.

That makes the type of inventory variable. The rationale for the old design was that we always wanted the inventory in a file because that allows one inventory fetch to cause many file downloads (for backfill).

Are you saying that you want to merge the inventory and download steps?

Reply:
I just wanted to have it be more flexible, I have been effectively merging the inventory and download steps while I was playing around with it. I didn't think what I added limited the ability for one inventory fetch to have many downloads to a file.